src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,660 @@
+receives and displays reports. See the
+[code](https://glitch.com/edit/#!/reports-endpoint).
+
+{% endBanner %}
+
+## Use cases and report types
+
+{% Banner 'caution', 'body' %}
+
+[Network Error Logging](https://w3c.github.io/network-error-logging/) isn't listed because it isn't supported in the new version of
+the API. Check the [migration guide](/reporting-api-migration/#network-error-logging) for details.
+
+{% endBanner %}
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Report type</th>
+        <th>Example of a situation where a report would be generated</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP">CSP</a> violation (Level 3 only)</td>
+        <td>You've set a Content-Security-Policy on your site, but one of your pages is trying to load a script that's not allowed by your CSP.</td>
+      </tr>
+      <tr>
+        <td><a href="https://web.dev/why-coop-coep/#coop">COOP</a> violation</td>
+        <td>You've set a Cross-Origin-Opener-Policy on your site, but a cross-origin window is trying to interact directly with your document.</td>
+      </tr>
+      <tr>
+        <td><a href="https://web.dev/why-coop-coep/#coep">COEP</a> violation</td>
+        <td>You've set a Cross-Origin-Embedder-Policy on your site, but a page on your site includes a cross-origin iframe that has not opted into being loaded by cross-origin documents.</td>
+      </tr>
+      <tr>
+        <td>Document Policy violation</td>
+        <td>Your site has a document policy that prevents usage of <code>document.write</code>, but a script on your page tries to use <code>document.write</code>.</td>
+      </tr>
+      <tr>
+        <td>Deprecation warning</td>
+        <td>Your site is using an API that is deprecated, directly or via a top-level third-party script.
+        </td>
+      </tr>
+      <tr>
+        <td>Intervention</td>
+        <td>Your site is trying to do something that the browser decides not to honor, for security, performance or user experience reasons. Example in Chrome: your site uses <a href="https://developers.google.com/web/updates/2016/08/removing-document-write"><code>document.write</code> on slow networks</a> or uses <code>navigator.vibrate</code> in a cross-origin frame that the user hasn't interacted with yet.</td>
+      </tr>
+      <tr>
+        <td>Crash</td>
+        <td>The browser crashes while your site is open.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+{% Aside %}
+**[Permissions
+policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy)
+(formerly feature policy)** violation reports may be supported by default in the future.
+Right now, they're experimental. One example situation where such reports would be
+generated: if your site has a permission policy to prevent microphone usage, and a
+script is requesting audio input.
```

Comment:
```suggestion
Right now, they're experimental. One example situation where such reports would be
generated is if your site has a permission policy to prevent microphone usage, and a
script is requesting audio input.
```